src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/shared/containers/ProcessStep.tsx

```diff
@@ -60,6 +61,11 @@ const ProcessStepComponent = (props) => {
               validations={validations}
               textResources={textResources}
             />
+            {instance?.status?.substatus &&
+            <AltinnSubstatusPaper
+              label={getTextResourceByKey(instance.status.substatus.label, textResources)}
+              description={getTextResourceByKey(instance.status.substatus.description, textResources)}
```

Comment:
there might not always be a description, how will this logic handle that?

Reply:
I have the same question. I would prefer if we could avoid conditional rendering, but if an empty description takes up space on the page we should consider it.

Reply:
Good question. Updated so that AltinnSubstatusPaper can handle an empty description without looking weird.